load_questions.dart

```diff
@@ -35,7 +38,7 @@ void main() async {
 
   for (final question in questions) {
```

Comment:
Would you be able to add a sort of polling/delay after creating the attribute and creating the document since attributes aren't ready right away. When I ran the load script, I got the following error:

```
$ dart ./load_questions.dart
Unhandled exception:
AppwriteException: document_invalid_structure, Invalid document structure: Unknown attribute: "question" (400)
#0      ClientMixin.prepareResponse (package:dart_appwrite/src/client_mixin.dart:73:9)
#1      ClientIO.call (package:dart_appwrite/src/client_io.dart:220:14)
<asynchronous suspension>
#2      Databases.createDocument (package:dart_appwrite/services/databases.dart:599:17)
<asynchronous suspension>
#3      main (file:///Users/steven/Projects/appwrite-demo-quiz-with-flutter/load_questions.dart:41:5)
<asynchronous suspension>
```

Reply:
Yeah. May be try a do-while loop. It’s usually efficient and sequential. Lemme work on it.

Reply:
Added 2 second delay. Tested locally and it works.